Symfony/CS/Fixer/Contrib/StrictFixer.php

```diff
@@ -55,13 +55,4 @@ public function getDescription()
     {
         return 'Comparison should be strict. Warning! This could change code behavior.';
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getPriority()
-    {
-        // should be run after the StandardizeNotEqualFixer
-        return -10;
```

Comment:
actually no need for this, as `<>` and `!=` are _THE SAME_ token kind for tokenizer :/